tdlibjson_wrapper/tdlibQt/models/singletons/UsersModel.cpp

```diff
@@ -37,7 +37,7 @@ UsersModel *UsersModel::instance()
 QString UsersModel::getChatTitle(const qint64 chatId)
 {
     if (!m_chats.contains(chatId))
-        return tr("Uknown chat");
+        return QObject::tr("Uknown chat");
```

Comment:
What is the reason for those tr() => QObject::tr() changes?

Reply:
I just try to call these string to be translated with QObject:: i not tested because i have some problems to install sailfish sdk. So i need your help to include these string be translated.

Reply:
I never worked with translations but looks like this change will not help. Only source files from depecher itself not tdlibjson_wrapper are scanned by lupdate.

Reply:
Maybe is missing something, but QObject::tr work to include some translation from files.cpp

Reply:
Fixed in https://github.com/elros34/depecher/commit/886d319730bd49909801fd5c8787bf673c0f1241

Reply:
wow @elros34 I'm proud of you. Thank you.